src/wallet/qi-hdwallet.ts

```diff
@@ -1353,11 +1353,11 @@ export class QiHDWallet extends AbstractHDWallet {
         const paymentCodeInfoArray = this._paymentCodeSendAddressMap.get(receiverPaymentCode);
         const lastIndex = this._findLastUsedIndex(paymentCodeInfoArray, account, zone);
 
-        let addrIndex = lastIndex;
+        let addrIndex = lastIndex + 1;
```

Comment:
Is the default value of `lastIndex` (aka the first time wallet calls this method) is the default value 0 or -1? If the answer is 0 then we should not be adding 1 here. The starting index should be 0.